javav2/usecases/pam_source_files/src/main/java/com/example/photo/services/S3Service.java

```diff
@@ -153,59 +164,68 @@ public String putManifest(String manifest) {
         UUID uuid = UUID.randomUUID();
         String key = uuid + ".csv";
         getClient().putObject(
-            PutObjectRequest.builder()
-                .bucket(PhotoApplicationResources.MANIFEST_BUCKET)
-                .key(key)
-                .build(),
-            RequestBody.fromString(manifest));
+                PutObjectRequest.builder()
+                        .bucket(PhotoApplicationResources.WORKING_BUCKET)
+                        .key(key)
+                        .build(),
+                RequestBody.fromString(manifest));
         return key;
     }
 
-    public String startRestore(String manifestArn, List<String> labels) {
-        String date = new SimpleDateFormat("yyyy-MM-dd").format(Calendar.getInstance().getTime());
-        String label = Strings.join(labels, '-');
+    public String startRestore(String manifestArn, List<String> labels, String eTag, String accountId, String iamRoleArn) {
+        String uuid = java.util.UUID.randomUUID().toString();
+        String reportBucketName = "arn:aws:s3:::" +PhotoApplicationResources.WORKING_BUCKET;
```

Comment:
This is an ARN, not a name, let's fix the variable name

Reply:
closed this method.